shared-scripts.sh

```diff
@@ -275,3 +275,184 @@ function _scanArtifacts {
         exit 1
     fi
 }
+
+function _putCredsInProfile {
+    mkdir -p ~/.aws
+    touch ~/.aws/config ~/.aws/credentials 
+    python3 codebuild_specs/sh-files/aws-configure-credentials.py
+}
+
+function _installIntegTestsDependencies {
+    apt-get update
+    apt-get install -y sudo
+    sudo apt-get install -y tcl
+    sudo apt-get install -y expect
+    sudo apt-get install -y zip
+    sudo apt-get install -y lsof
+    sudo apt-get install -y python3 python3-pip libpython3-dev
+    sudo apt-get install -y jq
+    sudo apt-get install -y libgbm-dev #
+    pip install awscli
```

Comment:
and we definitely have awscli installed, but maybe this is cliv1? either way if it isn't a lot of effort maybe try to comment out some of the known installations from our dockerfile https://github.com/aws-amplify/amplify-cli/blob/codebuild-image/latest/Dockerfile